api.go

```diff
@@ -471,19 +473,24 @@ func StartContainer(podID, containerID string) (*Container, error) {
 		return nil, err
 	}
 
+	cImpl, ok := c.(*Container)
```

Comment:
Not needed if we agree that `fetchContainer()` should not return `VCContainer` type.